backend/pkg/notification/notifications.go

```diff
@@ -688,77 +728,15 @@ func queueEmailNotifications(notificationsByUserID map[uint64]map[types.EventNam
 				}
 				//nolint:gosec // this is a static string
 				msg.Body += template.HTML(fmt.Sprintf("%s<br>====<br><br>", types.EventLabel[event_title]))
-				unsubURL := "https://" + utils.Config.Frontend.SiteDomain + "/notifications/unsubscribe"
-				for i, n := range ns {
+				i := 0
+				for _, n := range ns {
 					// Find all unique notification titles for the subject
 					title := n.GetTitle()
 					if _, ok := notificationTitlesMap[title]; !ok {
 						notificationTitlesMap[title] = true
 						notificationTitles = append(notificationTitles, title)
 					}
```

Comment:
The direct unsubscribe feature as implemented now does not work since we group events for various subscriptions inside a single email. Clicking the existing unsubscribe link will only unsubscribe from a single subscription included in the events.